pandas/tests/series/test_constructors.py

```diff
@@ -465,7 +465,7 @@ def test_constructor_index_mismatch(self, input):
         # test that construction of a Series with an index of different length
         # raises an error
         msg = 'Length of passed values is 3, index implies 4'
-        with pytest.raises(ValueError, message=msg):
+        with pytest.raises(ValueError, match=msg):
```

Comment:
I think there might have been a confusion between `message` and `match` here: message is the error printed on failure, `match` is the error message we check in the caught exception.

Reply:
That does appear to be the case.  Good catch!